admin/javascript/src/silverstripe-backend.js

```diff
@@ -58,7 +58,7 @@ class SilverStripeBackend {
    * Both `payloadFormat` and `responseFormat` can use the following shortcuts for their
    * corresponding mime types:
    *
-   *   - urlencoded: application/x-www-form-url-encoded
+   *   - urlencoded: application/x-www-form-urlencoded
```

Comment:
No, that me causing Ingo a lot of pain. Sorry!
On Mon, 18 Apr 2016 at 8:59 AM, Ingo Schommer <notifications@github.com>
wrote:

> In admin/javascript/src/silverstripe-backend.js
> <https://github.com/silverstripe/silverstripe-framework/pull/5316#discussion_r59992621>
> :
>
> > @@ -58,7 +58,7 @@ class SilverStripeBackend {
> >     * Both `payloadFormat` and `responseFormat` can use the following shortcuts for their
> >     * corresponding mime types:
> >     *
> > -   *   - urlencoded: application/x-www-form-url-encoded
> > +   *   - urlencoded: application/x-www-form-urlencoded
>
> @sminnee <https://github.com/sminnee> Just checking that you didn't write
> application/x-www-form-url-encoded on purpose? The last dash is wrong,
> and it causes data to be interpreted differently by CGI (doesn't end up in
> $_POST)
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/silverstripe/silverstripe-framework/pull/5316/files/ba059b30c96588431b3fd7e407047dab24568e41..57f94e6794f676df98e71caa9561d966ae22e558#r59992621>
>
-- 
Sam Minnée
CEO
SilverStripe Limited